services/bots/src/github-webhook/handlers/debug_info.ts

```diff
@@ -0,0 +1,23 @@
+import { IssuesLabeledEvent } from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+const COMMENTS = {
+    'cpu': ' - Start a new issue, choose profiler as the integration\n - Include a callgrind file from the <https://www.home-assistant.io/integrations/profiler/> `profiler.start` service\n - Include a `py-spy` from <https://community.home-assistant.io/t/instructions-to-install-py-spy-on-haos/480473>. (be sure to zip them up before posting as GitHub mutates svg files)\n - If memory is leaking, include 4 hours of RAW logs (`Settings -> System -> Logs -> Download`) from the <https://www.home-assistant.io/integrations/profiler/> `profiler.start_log_objects` service with the default settings'
```

Comment:
Use [my](https://my.home-assistant.io/create-link/) links when you tell the user to navigate inside Home Assistant.

Reply:
> Start a new issue, choose profiler as the integration
That part does not make sense to me, they have an issue already.

If they really need to create a separate one, give them a link that does that and add the correct integration for them.

Like <https://github.com/home-assistant/core/issues/new?assignees=&labels=&projects=&template=bug_report.yml&integration_name=profiler&integration_link=https%3A%2F%2Fwww.home-assistant.io%2Fintegrations%2Fprofiler>

Reply:
Yea this is what @bdraco came up with. Can you maybe elaborate on the new issue part? Most of these issues are created without any integration link, so we should be able to add profiler label ourselves and add the cpu label to add these instructions